RELEASES.md

```diff
@@ -6,6 +6,7 @@ Version 0.9.5 (2023-09-27)
 * New driving model
   * Improved navigate on openpilot performance using navigation instructions as an additional model input
 * Hyundai Azera 2022 support thanks to sunnyhaibin!
+* Hyundai Custin (Taiwan only) 2023 support thanks to sunnyhaibin and Saber422!
```

Comment:
Fine to remove `Taiwan only` if it's not sold in the US https://github.com/commaai/openpilot/pull/29876#discussion_r1323871685